Cargo.lock

```diff
@@ -1061,7 +1061,7 @@ dependencies = [
  "backoff",
  "derivative",
  "futures",
- "hashbrown 0.14.3",
+ "hashbrown 0.14.5",
```

Comment:
let's also revert the cargo.lock changes -- we don't mean to pick up dependency changes with this crd change.

Reply:
Synced lockfile w/ `linkerd/linkerd2@main`